docs/product-getting-started-guide/src/main/asciidoc/test-suspend-proc.adoc

```diff
@@ -0,0 +1,46 @@
+[id='testing_suspend-proc']
+= Testing the driver license suspension scenario
+
+The following chapter instructs you how to test the driver license suspension scenario using the data that you specified when you set the Driver License Suspension rules and actions.
```

Comment:
See my comment in the Creating the Violation data object data fields section, about intro statements.

Reply:
done